maint/Makefile.PL.include

```diff
@@ -0,0 +1,29 @@
+BEGIN { -e 'Distar' or system("git clone git://git.shadowcat.co.uk/p5sagit/Distar.git") }
+use lib 'Distar/lib';
+use Distar 0.001;
+
+author 'Caelum: Rafael Kitover <rkitover@cpan.org>';
+
+manifest_include 't/backcompat/0.04006/lib' => qr/.*\.pm$/;
+manifest_include 't/bin' => qr/.*/;
+
+use FindBin;
+use lib "$FindBin::Bin/lib";
+require DBIx::Class::Schema::Loader::Optional::Dependencies;
+my $optdeps = 'DBIx::Class::Schema::Loader::Optional::Dependencies';
+my @req_groups = keys %{ $optdeps->req_group_list };
+my @rdbms_groups = grep { /rdbms/ } @req_groups;
+my @other_groups = grep { !/rdbms/ } @req_groups;
+our (%dev_requires, %runtime_suggests);
+%dev_requires = %{ $optdeps->modreq_list_for(\@other_groups) };
+%runtime_suggests = %{ $optdeps->modreq_list_for(\@rdbms_groups) };
```

Comment:
We don't currently suggest any of the RDBMS-specific modules, and I'd like to keep actual behaviour changes to a minimum in this PR.  If we want to further tweak the deps, that should be a separate discussion.

Reply:
Agreed, changed.